scripts/src/metrics/metrics.py

```diff
@@ -14,7 +14,13 @@
 from pullrequest import prepare_pr_comment as pr_comment
 
 file_pattern = re.compile(r"charts/([\w-]+)/([\w-]+)/([\w\.-]+)/([\w\.-]+)/.*")
-ignore_users=["zonggen","mmulholla","dperaza4dustbit","openshift-helm-charts-bot","baijum","tisutisu","rhrivero"]
+ignore_users=["zonggen","mmulholla","dperaza4dustbit","openshift-helm-charts-bot","baijum","tisutisu","rhrivero","Kartikey-star"]
+pr_submission="PR Submission v1.0"
+pr_merged="PR Merged v1.0"
+pr_outcome="PR Outcome v1.0"
+id_prefix="helm-test-metric-pr"
```

Comment:
Might be better if we select the prefix based on the environment - so if  host repo is not openshift/charts use "helm-test-metric-pr" otherwise use "helm-metric-pr"